### DISCUSSION TITLE
Loading Images from Local Files in UI Cards

### DISCUSSION THREAD

**Discussion by johannesmphaka** [h2oai/wave#2324] [Q&A] [answered]
howzit guys. 

Im working on a project where I'm creating cards with information displayed on them. Currently, I'm using the ui.wide_info_card() function to create these cards. However, I'm facing an issue with loading images from my local files onto these cards.

q.page['example'] = ui.wide_info_card(
    box='1 1 4 5',
    name='info_card',
    title='Info Card',
    subtitle='Subtitle',
    caption='''
Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quia aliquam maxime quos facere
necessitatibus tempore eum odio, qui illum. Repellat modi dolor facilis odio ex possimus
ratione voluptate pariatur cupiditate, quidem quaerat sapiente exercitationem in omnis nulla
maiores consequatur dolores illo inventore quae obcaecati culpa totam corporis! Repudiandae, nostrum repellendus.
''',
    category='Category',
    label='Click me',
    image='plot.png'
)

**Comment by mturoci** [comment, marked as answer]
See https://wave.h2o.ai/docs/files.

**Reply by johannesmphaka** [reply]
It worked, thanks.
